narps_open/pipelines/matlabbatch_UK24.m

```diff
@@ -0,0 +1,292 @@
+% volume that are part of white matter (as computed using the segmentation,
+% threshold>0.5?)
+
+% --> Save as reg_wm.txt
+
+% CSF signal nuisance: average value of voxels in th functional
+% volume that are part of white matter (as computed using the segmentation,
+% threshold>0.5?)
+
+% --> Save as reg_csf.txt
+
+% "independent_vars_first_level": "The GLM design matrix for the 1st level 
+% analysis consited of 17 explicitly specified regressors per run. Of these
+% regressors, the first 8 related to experimental conditions, whereas the 
+% remaining 9 were nuisance regressors (6 head motion parameter regressors,
+% 2 tissue signal regressors, and 1 scrubbing regressor), none of which 
+% were entered as interactions. 
+% For the experimental conditions, we are 
+% interested in the parametric effect of gain and of loss as related to a 
+% mixed gambles task. Trial data, indicating the amount of possible gain 
+% and the amount of possible loss, as well as trial timing, trial durations
+% and reaction times were supplied for each run per subject. From this data
+% a block-task regressor was derived respectively for all trials where the 
+% gamble could result in a possible gain, in a possible loss, and in a 
+% no-loss-nor-gain situation (i.e. 3 block-task regressors). In each case 
+% of the possible gain/loss regressors, 2 parametric modulation regressors 
+% were added: 1 to model the parametric size of the possible gain/loss, and
+% 1 to model the parametric size of the reaction time. In the case of the 
+% no-loss-nor-gain block-task regressor a single extra parametric 
+% modulation regressor was added to model the parametric size of the 
+% reaction time. 
+% All parametric modulation regressors were mean centered 
+% before adding them to the design matrix for the GLM analysis, which 
+% implicitly (in SPM12) orthogonalises all parametric modulators in a 
+% series with reference to the first one. No time-modulation of conditions 
+% was included.\n\nThe canonical HRF basis (default SPM12 option) was used 
+% for convolution of the 8 conditions. SPM12 automatically also included a 
+% constant regressor in the design matrix, and filtered the data and design 
+% matrix using a discrete cosine basis set with a specified cutoff 
+% frequency of 128 Hz (default). This accounted for slow drifts in the data
+% , and no explicit drift regressors were included in the design matrix."
+
+% Note: the covariates for tissue signal regressors and scrubbing regressor
+% not implemented yet
+
+% 8 regresseurs : gain, gain_value, gain_RT, loss, loss_value, loss_RT, no gain no loss, RT no gain no loss
+% 9 nuisance : 6 head motion parameter regressors, 2 tissue signal regressors, and 1 scrubbing regressor
+
+matlabbatch{end+1}.spm.stats.fmri_spec.dir = '<UNDEFINED>';
+matlabbatch{end}.spm.stats.fmri_spec.timing.units = 'secs';
+matlabbatch{end}.spm.stats.fmri_spec.timing.RT = '<UNDEFINED>';
+matlabbatch{end}.spm.stats.fmri_spec.timing.fmri_t = 16;
```

Comment:
What is the purpose of these parameters (`fmri_t` and `fmri_t0`) ?

Reply:
This is something I should have removed from the matlabbatch as this is just the default value.

For info (as this might be useful for other parameters), each parameter of fMRI specification is described in this file: https://github.com/spm/spm12/blob/main/config/spm_cfg_fmri_spec.m (and in correspondant `spm_cfg_xxx` for each of the matlabbatch modules).